backend/src/services/AllowlistEntry.ts

```diff
@@ -48,9 +49,13 @@ function AllowlistEntryService(
         id,
       },
     });
+    if (!allowlistEntry) {
+      const msg = `allowlistEntry not found for id ${id}`;
```

Comment:
This wouldn't be 'not found', it would be 'failed to be created'

Reply:
but this is in deleteAllowlistEntry

Reply:
oh. i'm dumb. resolving this